scripts/ait_bootstrap.sh

```diff
@@ -70,34 +106,27 @@ EOM
 mkvirtualenv ait
 workon ait
 
-# Pull assets, config, and secrets from s3/sm
+# Pull assets, config, and secrets from s3/sm (NEEDS TO MOVE)
 mkdir -p $SETUP_DIR
-/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 sync s3://$CONFIG_BUCKET_NAME/ait/ $SETUP_DIR/
-/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 cp s3://"$CONFIG_BUCKET_NAME"/modules/openmct-static.tgz - | tar -xz -C /var/www/html
-
+/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 sync s3://$CONFIG_BUCKET_NAME/configs/ait/ $SETUP_DIR/
+/usr/local/aws-cli/v2/current/bin/aws --region $AWS_REGION s3 cp s3://"$CONFIG_BUCKET_NAME"/configs/modules/openmct-static.tgz - | tar -xz -C /var/www/html
```

Comment:
This doesn't need to happen in user-data anymore since these files are brought onto EFS as part of efs-bootstrap. We will probably want to copy/untar the openmct stuff off of EFS to the instance's /var/www/html though